synapse/cores/common.py

```diff
@@ -681,7 +650,48 @@ def initCoreModule(self, ctor, conf):
             logger.warning('mod load fail: %s %s' % (ctor,e))
             return False
 
+    def _synTagsVers0(self):
```

Comment:
docs are sorted alphabetically, and 'private' functions are not included.  More of a code-style thing; since over time we could have revisioning code growing in the middle of a file.  That is why I moved the model defs to the bottom of those files; so they revisions could just be added to the bottom over time.